### PR TITLE
docs: improve site docs

### PR DESCRIPTION
**Pull request by Parul1606**
## Related Issue
Closes issue #1661 

## Description

I have improved the site's docs. 

## Screenshots

image

image

image

image

image

image

image

### CODE REVIEW

File: components/AddYourProjectsGuide.tsx [hunk 1]
```diff
@@ -91,58 +104,124 @@ const option2Steps = [
...
+    text: "Perform your desired Changes and if you are contributing a project then you can follow the below command",
     code: "",
   },
   {
     icon: <PiArrowFatLineRightFill size={18} color="white" />,
-    text: "Add your changes",
-    code: "git add .",
+    text: "Add following code to end of src/DB/projects.json",
+    code: codeString,
+  },
```
Comment (priyankarpal) [inline components/AddYourProjectsGuide.tsx:177]: remove the code. we don't need to add `json` code here

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: components/AddYourProjectsGuide.tsx (hunk 1 above)
Comment (Parul1606) [reply components/AddYourProjectsGuide.tsx:177]: It will be done by today

Comment (Parul1606) [reply components/AddYourProjectsGuide.tsx:176]: image

@priyankarpal while editing your highlighted issue is not seen. What do you think I should do now?

Comment (Parul1606) [reply components/AddYourProjectsGuide.tsx]: Is it okay now @priyankarpal?

**Comment by priyankarpal** [comment]
Hey @Parul1606  ! 👋

For secure collaboration, remember to check out GitHub's commit signature verification feature. It helps ensure the authenticity of commits and prevents tampering.

Learn how it works: [GitHub Docs](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)

Happy coding! 😄

**Pull request merged**